### PR TITLE
updated luminesce sdk requirement

### PR DESCRIPTION
**Pull request by kr0bfb**
chronjob build failing, luminesce sdk version has been pinpointed as the cause. Updating the the value and conducting a test run.

I tried running this locally to no avail, was told that I should just open a pr with the changes to see if the build passes

Success pending...

### CODE REVIEW

**Review by slemasne — APPROVED** [review]

File: .github/workflows/build-and-test.yaml [hunk 1]
```diff
@@ -63,7 +63,7 @@ jobs:
       - name: Run tests
         id: run_tests
         env:
-          FBN_BASE_API_URL: ${{ secrets.URI }}
+          FBN_BASE_API_URL: ${{ secrets.FBN_BASE_API_URL }}
```
Comment (adamMQM) [inline .github/workflows/build-and-test.yaml:66]: Can you please change this back, then we can re-run tests

**Review by adamMQM — APPROVED** [review]

**Pull request merged**